docs/dev-guide/contributing/building.rst

```diff
@@ -272,19 +280,32 @@ Run the build script with the following syntax::
 
     $ ./build-all.py <name of yaml file> [options]
 
-For example, to perform a test build of the 2.6-dev release as specified in
-``pulp_packaging/ci/config/releases/2.6-dev.py`` where the results are not pushed to
+For example, to perform a test build of the 2.6-build release as specified in
+``pulp_packaging/ci/config/releases/2.6-build.py`` where the results are not pushed to
 fedorapeople::
 
     $ ./build-all.py 2.6-dev --disable-push
 
+Reconcile Redmine Issues
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Before starting a release build, ensure that there are no issues in a MODIFIED state without
```

Comment:
Do we want to use double backtics to escape the state names like
```
``MODIFIED``
```

Whatever you decide is fine.